groups/models.py

```diff
@@ -161,6 +161,20 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def subscribers(self):
```

Comment:
I mean, really, I'd suggest putting it back where it came from.  I don't think there's anything wrong with having it there, and it has benefits of readability and easy semantics.  Code that only a view uses going in that view = good.  As I asked, is the method ever going to be used elsewhere?